blocks/ifus/ifus.css

```diff
@@ -0,0 +1,76 @@
+.ifu {
+    display: flex;
+    flex-flow: row nowrap;
+    justify-content: center;
+    padding: 2rem 0 0;
+    max-width: 1300px;
+    margin: auto;
+}
+
+.ifu-result {
+    width: 100%;
+    padding: 2rem 0 0;
+    max-width: 1300px;
+    margin: auto;
+}
+
+.ifu-result.no-result {
+    visibility: hidden;
+    padding: 0;
+}
+
+@media (max-width: 1024px) {
+    .ifu {
+        flex-direction: column;
+    }
+}
+
+.ifu > div {
+    padding: 15px 20px;
+    width: 100%;
+}
+
+.ifu-result > div > div {
+    padding: 0 1rem 2rem;
+    border-bottom: 1px solid #75787b;
```

Comment:
should probably use local var referencing `--mt-global-color-accent-quaternary`

Reply:
Done